crates/matrix-sdk/src/widget_api/messages/capabilities.rs

```diff
@@ -144,36 +79,87 @@ impl<'de> Deserialize<'de> for Options {
             if capability.starts_with("org.matrix.msc2762.m.send.event") {
                 let cap_split: Vec<&str> = capability.split(":").collect();
                 if cap_split.len() > 1 {
-                    send_room_event.push(serde_json::from_str(cap_split[1]).unwrap());
+                    capbilities.send_room_event.push(serde_json::from_str(cap_split[1]).unwrap());
```

Comment:
- There seems to be a typo `capbilities` -> `capabilities`.
- `capability.split(":").at(1).map(|...|)` could be used instead.

Reply:
fixed here: https://github.com/toger5/matrix-rust-sdk/pull/16/commits/5d073610d6026b3f6f05c456139e34e5a323348f